client/container.go

```diff
@@ -20,8 +20,6 @@ import (
 
 // ContainerPutPrm groups parameters of PutContainer operation.
 type ContainerPutPrm struct {
-	prmSession
```

Comment:
`SetSessionToken` was not used anywhere?

Reply:
The problem is that `ContainerPutPrm` provides two ways to set session token: 
- via `prm.SetSessionToken`
- via `cnr.SetSessionToken` which provided into `prm`.

I suggested to keep only one way to set container session token. Removing setter from `cnr` seems less intuitive and a bit more complicated for me, because this structure is used much more in the code of NeoFS Node and other apps. 

We can keep both setters as well, but then client should check both of this session token sources in some order, I think.

Reply:
If some code already uses `SetSessionToken` we can break its logic. We can do it safely for example: 
```
// SetSessionToken ...
// Deprecated: write session token into container structure.
func (x *ContainerPutPrm) SetSessionToken(t session.Token) {
	if x.cnrSet {
		x.cnr.SetSessionToken(&t)
	} else {
		x.prmSession.SetSessionToken(t)
	}
}

// SetContainer sets structured information about new NeoFS container.
// Required parameter.
func (x *ContainerPutPrm) SetContainer(cnr container.Container) {
	if x.tokenSessionSet {
		cnr.SetSessionToken(&x.tokenSession)
	}

	x.cnr = cnr
	x.cnrSet = true
}
```

Reply:
Lets remove prm method.

Reply:
Already done in this PR by removing `prmSession` from `ContainerPutPrm`, or you mean something else?

By the way, `SetEACL` and `DeleteContainer` have only one way to set session token now, thus they will keep using `prmSession`.

Reply:
Yeah, I see. Just thought that method removal will break the compatibility, but we don't have a release version yet.